mlir/test/rocmlir-driver/populate_host_convert.mlir

```diff
@@ -1,13 +1,13 @@
 // RUN: rocmlir-gen --arch %arch -p -ph -pr | FileCheck %s --check-prefix=F32
 // RUN: rocmlir-gen --arch %arch -p -ph -pr -t f16 | FileCheck %s --check-prefixes=F16,CHECK
 // RUN: rocmlir-gen --arch %arch -p -ph -pr -t bf16 | FileCheck %s --check-prefixes=BF16,CHECK
-// RUN: rocmlir-gen --arch %arch -p -ph -pr -t i8 | FileCheck %s --check-prefixes=I8,CHECK
```

Comment:
Why is `CHECK` removed here?

Reply:
It's used to check that a conversion function exists. We used to convert i32 to f32 and print. I've changed it to directly print out i32. So there shouldn't be a conversion function, just like f32.

Reply:
Ah, right, now I'm seeing how that test is structured. Makes sense.